### PR TITLE
updated _includes/getting-stareted.html

### LINKED ISSUE #517: Copy text gets hidden
**Description**
In the Getting Started portion of the website, the tooltip for the copy clipboard is getting cut off

**Expected Behavior**
The complete tooltip should be visible.

**Screenshots**

https://user-images.githubusercontent.com/53532328/129491907-3d9fc663-6192-4ba4-ae04-63d37ffe0d6a.mov

**Enviroment:**
 - OS: MacOs
 - Browser:  Chrome, Safari
 

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by ANURAG-PATHAK**
Signed-off-by: ANURAG-PATHAK <pathakanurag605@gmail.com>

**Description**

This PR fixes #517 #500 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by NikhilSharma03 — APPROVED** [review]
LGTM :+1:

**Review by warunicorn19 — APPROVED** [review]
@ANURAG-PATHAK changes LGTM🎉! Just something to keep in mind for next time, if you could please make seperate PRs for seperate issues, putting everything together in one PR makes it difficult to the review the code. 

Also, Thanks for these changes, good work 😃.

**Comment by ANURAG-PATHAK** [comment]
> @ANURAG-PATHAK changes LGTM🎉! Just something to keep in mind for next time, if you could please make seperate PRs for seperate issues, putting everything together in one PR makes it difficult to the review the code.
> 
> Also, Thanks for these changes, good work 😃.

Sure, I will take care of it next time, and thank you for your help and guidance.

**Review by Jashpatel1 — COMMENTED** [review]
Looks good. One more change if @leecalcote can confirm. Do we change the program name from CommunityBridge to LFX on the programs page as well?

**Comment by warunicorn19** [comment]
@Jashpatel1 if I'm not wrong, aren't we looking for new designs for that page?

**Comment by ANURAG-PATHAK** [comment]
> Looks good. One more change if @leecalcote can confirm. Do we change the program name from CommunityBridge to LFX on the programs page as well?
Screenshot 2021-08-22 at 11 06 00 AM

I was thinking the same but @warunicorn19 told that they will be redesigning the whole page

**Comment by ANURAG-PATHAK** [comment]
I would have to commit a change because I deleted the community bridge icon so if my pr will be merged the site will look like this 
Screenshot 2021-08-22 at 12 16 56 PM

that is why I think I should make this LFX then we can think of redesigning  the page as well so should I commit the change in this pr please approve

**Comment by ANURAG-PATHAK** [comment]
Screenshot 2021-08-22 at 12 42 19 PM



**this would be the new look after my commit**
should I make a commit for the same? 
@warunicorn19 @Jashpatel1

**Comment by leecalcote** [comment]
@ANURAG-PATHAK a request: the white versions of these logos are terribly unattractive. Do you mind finding SVGs of each with color, so that we can assess whether shifting to colorized version is a bit more aesthetically pleasing?

**Comment by warunicorn19** [comment]
Maybe we could try using something like https://iconscout.com/icon/linux-foundation-3521550

**Comment by ANURAG-PATHAK** [comment]
> @ANURAG-PATHAK a request: the white versions of these logos are terribly unattractive. Do you mind finding SVGs of each with color, so that we can assess whether shifting to colorized version is a bit more aesthetically pleasing?

Yes I can make a prototype using coloured SVGs, but I think we need to redesign this whole page as these large icons are making it a bit unattractive .

**Comment by ANURAG-PATHAK** [comment]
@leecalcote , These are the updated changes 


Screenshot 2021-08-25 at 2 02 07 AM









Screenshot 2021-08-25 at 2 02 35 AM

**Comment by leecalcote** [comment]
@ANURAG-PATHAK thanks. That's not the MLH logo, no?

**Comment by ANURAG-PATHAK** [comment]
> @ANURAG-PATHAK thanks. That's not the MLH logo, no?

there are two logos one is for mlh and other is of mlh fellowship

**Comment by leecalcote** [comment]
Thanks, @ANURAG-PATHAK. Almost there?

**Comment by ANURAG-PATHAK** [comment]
> Thanks, @ANURAG-PATHAK. Almost there?
yes, tell me if I need to change anything else

**Comment by ANURAG-PATHAK** [comment]
> Thanks, @ANURAG-PATHAK. Almost there?

so which logo you want on the website mlh one or mlh-fellowship one

**Comment by warunicorn19** [comment]
@ANURAG-PATHAK the fellowship one works for now, also if you get rid of the merge conflict.

**Comment by warunicorn19** [comment]
@ANURAG-PATHAK could you please put in the LFX one as well?

**Comment by warunicorn19** [comment]
@Jashpatel1 should we go with the coloured ones?

**Comment by ANURAG-PATHAK** [comment]
> @ANURAG-PATHAK could you please put in the LFX one as well?

sorry @warunicorn19 what LFX ? please elaborate 😊

**Comment by warunicorn19** [comment]
Screenshot 2021-09-02 at 3 00 02 PM

**Comment by ANURAG-PATHAK** [comment]
so if you can confirm I can push the changes with coloured logo and LFX also or should I go with this one?

**Comment by leecalcote** [comment]
Pulling in the big gun, @Nikhil-Ladha here.

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```
Comment (Nikhil-Ladha) [inline .idea/.gitignore:1]: Remove all the `.idea` files from the commit.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Make a commit with the colored logos.
Change `Community Bridge` to `LFX` and also update the filename such that the page path reflects the same.
Use the MLH same as used [here](https://layer5.io/careers/programs).

**Comment by ANURAG-PATHAK** [comment]
due to some errors, I pulled another PR #544 
sorry for the inconvenience 😔😔

**Comment by Nikhil-Ladha** [comment]
Closing this PR based on the above comment.

**Pull request closed without merging**